plaso/cli/helpers/elastic_output.py

```diff
@@ -60,6 +60,12 @@ def AddArguments(cls, argument_group):
         default=cls._DEFAULT_RAW_FIELDS, help=(
             'Export string fields that will not be analyzed by Lucene.'))
 
+    argument_group.add_argument(
+        '--additional', '--additional-fields', dest='additional_fields',
+        action='store', default='', help=(
+            'JSON string with a dict of fields and values to add to each '
```

Comment:
Please add an example or documentation of what value is expected here

Reply:
Also why JSON and not just a string of field names?

Reply:
well, this is a draft PR, and will not be set to a PR until I've done some testing on the TS side, and written the code there.

Then this will all be better explained, I just added this here so that you would see what the intention was, and how it looks